MdeModulePkg/Library/MemoryTypeInfoSecVarCheckLib/MemoryTypeInfoSecVarCheckLib.c

```diff
@@ -84,8 +95,13 @@ MemoryTypeInfoVarCheckHandler (
   if (DataSize != EFI_MEMORY_TYPE_INFORMATION_VARIABLE_SIZE) {
     DEBUG ((
       DEBUG_ERROR,
+<<<<<<< HEAD

```

Comment:
This does not seem right?

Reply:
Yeah, the merge conflict markers were left in. Fortunately, this will need to be fixed to pass compilation.

Reply:
Thanks for finding that.